doc/docs/2.3.x/deploy-manage/deploy/loki.md

```diff
@@ -74,6 +74,15 @@ loki-stack:
        
          ![Container logs](../images/grafana_user_logs.png)
 
+## Using Loki in Another Namespace
```

Comment:
Should this actually be in the `master` directory instead of the `2.3.x` directory?  I don't actually know, just assuming.

Reply:
Oh yeah I think you're right, good catch!